generics/helpers/utils.js

```diff
@@ -732,6 +732,39 @@ function getTranslatedData(data, translateData) {
 	})
 	return data
 }
+
+/**
+ * Function to calculate the end date based on a start date and duration string.
+ * @param {string} createdDate - The start date in ISO format (e.g., '2024-12-05T00:00:00.000Z').
+ * @param {string} durationString - Duration in the format "1 week", "2 weeks", "1 month", "1 year", etc.
+ * @returns {string} - The calculated end date in ISO format
+ */
+function calculateEndDate(createdDate, durationString) {
+	const startDate = moment(createdDate) // Parse the start date
+	console.log(`Calculating end date`, startDate)
```

Comment:
@praveenKDass please remove this console

Reply:
Removed @VISHNUDAS-tunerlabs